apps/src/sites/studio/pages/blocks/edit.js

```diff
@@ -167,7 +167,10 @@ function updateBlockPreview() {
   });
   const blocksDom = parseElement(`<block type="${blockName}" />`);
   Blockly.mainBlockSpace.clear();
-  Blockly.cdoUtils.loadBlocksToWorkspace(Blockly.mainBlockSpace, blocksDom);
+  Blockly.cdoUtils.loadBlocksToWorkspace(
+    Blockly.mainBlockSpace,
+    Blockly.Xml.domToText(blocksDom)
```

Comment:
similar question here, do we need both `parseElement` and `domToText`?

Reply:
we do not!